frontend/src/js/controllers/authCtrl.js

```diff
@@ -257,9 +257,9 @@
                             if (token_valid) {
                                 vm.FormError = "this link has been already used or expired.";
                             } else if (password1_valid) {
-                                vm.FormError = response.data.new_password1[0] + " " + response.data.new_password1[1];
```

Comment:
@anweshknayak : There are some cases when error response is something like this
```
new_password1 {
    0: "error 1",
    1: "error 2" 
}
```
So solve it by applying loop method till the length of `new_password1`